libnfc/CMakeLists.txt

```diff
@@ -26,23 +29,23 @@ IF(UART_REQUIRED)
 ENDIF(UART_REQUIRED)
 
 IF(I2C_REQUIRED)
-  IF(WIN32)
-    # Windows is not supported at the moment
+  IF(NOT UNIX OR APPLE)
```

Comment:
May I suggest to reverse the test and branches, to ease reading ?
if (unix or apple) ... set ON... else... set OFF